src/main/java/io/jenkins/plugins/checks/github/SCMFacade.java

```diff
@@ -19,6 +19,7 @@
 import jenkins.scm.api.SCMRevision;
 import jenkins.scm.api.SCMSource;
 import jenkins.triggers.SCMTriggerItem;
+import jline.internal.Nullable;
```

Comment:
this looks like a strange annotation, normally you would use `CheckForNull` from spotbugs

Reply:
Yes, that's why Spotbugs still does find a problem...